### PR TITLE
Feature/percentage of wins

### PR DESCRIPTION
**Pull request by noahdurbin**
## Type Of Change
- [X] New Feature
- [ ] Debugging
- [ ] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change

So in this branch I did a couplthings and while all my methods are working the way that they should as far as I can tell I am still stuck with getting tests to reflect that. when ran in isolation the tests pass just fine but when I try to run all the tests together it counts more objects than it should and I think it's because we are storing the game objects in a class variable which in theory will be fine since we aren't planning on creating our games more than once. I maybe wen overboard with the helper methods this time around but I do think that these methods ended up pretty readable as a result of the helper methods. Would love to hear what you think!

I also spent some time trying to figure out how to convert the date from a string into a usable date but wasn't quite able to figure out that puzzle so I may return to that at some point. As we have it right now it works but the date is not practically usable.

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before sucmitting PR
- [X] I assigned atleast 2 group members to review my PR

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -37,4 +39,53 @@ def self.lowest_total_score
       game.total_score
     end.min
   end
+
+  def self.game_count
+    GameFactory.all_games.length
+  end
+
+  def self.percentage_home_wins
+    (count_home_wins.to_f / game_count.to_f) * 100.00
```
Comment (zach-bergman) [inline lib/game.rb:48]: Awesome helpers, makes this really clean!

**Review by zach-bergman — APPROVED** [review]
Nice job, looked through everything and it looks great. Love the use of helper methods

File: lib/game.rb (hunk 1 above)
Comment (austincarrjones) [reply lib/game.rb:48]: Cool looks good, I guess some of these attributes will be removed per Kat's check-in suggestion of YAGNI (I'll have to do this for Teams too)

File: lib/game.rb [hunk 2]
```diff
@@ -37,4 +39,53 @@ def self.lowest_total_score
...
+      if game.away_win?
+        win_count += 1
+      end
+    end
+  end
+
+  def home_win?
+    if @home_goals > @away_goals
+      true
+    else
```
Comment (austincarrjones) [inline lib/game.rb:76]: *reminder to myself as well to remove the conditionals here when it's implied.

**Review by austincarrjones — APPROVED** [review]
Nice work! I'll be taking a peek here to see if these methods will apply to some of my list.

**Pull request merged**